README.md

```diff
@@ -16,16 +16,17 @@ It is also an application generator for building an application based on yaktor.
 
 ## Usage
 
-Typically, you'll want to
+Typically, you'll want to do something like
 
 ```
-$ docker run -it -v "$PWD":/app --rm yaktor/yaktor yaktor create myapp
+$ mkdir myapp && cd "$_"
```

Comment:
Most people will have to go search to find that `$_` is Bash for "most recent parameter".  Readability should outweigh DRY here.  I recommend `cd myapp`.